### ISSUE TITLE
Unable to remove friend

### ISSUE THREAD

**Issue by lmfardin** [EionRobb/skype4pidgin#576]
Version: 1.4

If I remove a friend on Pidgin and log on again, the friend is still there.
I have removed it many times, but it always gets back.
A workaround might be removing it straight from Skype.

**Comment by lmfardin** [comment]
Still unable to remove friends on Skype. Anyone else facing the same?

**Comment by zaksmok** [comment]
Same issue here. 
Debian 9
Pidgin 2.12.0 (libpurple 2.12.0)
skype4pidgin 1.4

**Comment by EionRobb** [comment]
Does it happen with the latest code in git?

**Comment by zaksmok** [comment]
Yes. I used latest .deb file

**Comment by EionRobb** [comment]
@zaksmok no, latest .deb isn't latest git.  You'll need to follow https://github.com/EionRobb/skype4pidgin/tree/master/skypeweb#compiling to build the latest git code to test

**Comment by zaksmok** [comment]
@EionRobb true, i used .deb for another plugin. I built skype plugin from latest git code and this issue happens + i can't add new friends. It doesn't add to the list + friend require shows after every restart
